backend/src/apps/slack/admin/content_report.py

```
@@ -28,12 +32,13 @@ class ReactionAlertAdmin(admin.ModelAdmin):
         "conversation__name",
         "message_ts",
         "report_type",
+        "source",
     )
 
     def has_add_permission(self, request):
```

Ruff (0.16.1)

[warning] Unused method argument: `request`

(ARG002)